### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by jburel**
Fix some typo in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ The following prerequisites are required for deploying a Jenkins devspace:
 
 The default deployment initializes a Jenkins server with a [predefined set of
 jobs](homes/jobs). The table below lists the job names, the Jenkins node labels
-they are associated to and a short description of their:
+they are associated to and a short description of the job:
```
Comment (mtbc) [inline README.md]: "associated *with*", perhaps

File: README.md [hunk 2]
```diff
@@ -239,30 +239,30 @@ Default packages:
...
- *  Uprade to 0.2.0:
+ *  Upgrade to 0.2.0:
 
-    If you made custom adjustments to the code and commited them, it is recomanded to reset changes.
+    If you made custom adjustments to the code and commited them, it is recommended to reset changes.
 
     Here are listed the most important changes:
 
-     * Compose configuration was splitted into a few different files depends on the platform
+     * Compose configuration was splitted into a few different files depending on the platform
```
Comment (mtbc) [inline README.md]: "splitted" to "split"

**Comment by mtbc** [comment]
These changes all look good to me. Couple of others suggested but not blocking this PR.

**Comment by aleksandra-tarkowska** [comment]
Could this PR be open against my branch, that will let us merge that PR with #50 ?

**Comment by jburel** [comment]
@aleksandra-tarkowska: I can do that

**Comment by jburel** [comment]
Closing this PR, PR https://github.com/aleksandra-tarkowska/devspace/pull/1 opened against @aleksandra-tarkowska's one

**Pull request closed without merging**